lib/web_ui/lib/src/engine/host_node.dart

```diff
@@ -187,3 +202,105 @@ class ElementHostNode implements HostNode {
   @override
   void appendAll(Iterable<DomNode> nodes) => nodes.forEach(append);
 }
+
+// Applies the required global CSS to an incoming [DomCSSStyleSheet] `sheet`.
+void applyGlobalCssRulesToSheet(
```

Comment:
We should apply the `color: red` rule here, so it only affects to the contents of the `flt-glass-pane`, and not the outer slots!

Reply:
Yeah, I think this ended up fixing that issue. Added to the description, thanks for the link!!